website/docs/cheatsheets/python/v1/data.md

```diff
@@ -251,6 +251,44 @@ config.run_config.data_references[data_ref.data_reference_name] = data_ref.to_co
 ## Create Dataset
 
 ### From local data
+You could create and register a dataset directly from a folder on your local machine. Note that `src_dir` must point to a **folder**, not file.
+:warning: Method `upload_directory`: This is an experimental method, and may change at any time. Please see https://aka.ms/azuremlexperimental for more information.
+```python
+# upload the data to datastore and create a FileDataset from it 
+folder_data = Dataset.File.upload_directory(src_dir="PATH/TO/FOLDER", target=(datastore, "SELF-DEFINED/PATH/ON/DATASTORE"))
+dataset = folder_data.register(workspace=ws, name="test_data")
+```
+
+### From a datastore
+The code snippet below shows how to create a Dataset of an entire folder given the relative path `local/test/` on `datastore`. Note that this also applies to creating a Dataset of a single file, which could be done using a relative path like `local/test/data.tsv`.
+```python
+import os
+from azureml.core import Dataset
+
+# create input dataset
+dataset_name = os.environ.get('ADL_DATA_SET', 'folder_data')
+folder_data = Dataset.File.from_files(path=(datastore, os.environ.get('ADL_DATA_RELATIVE_PATH', 'local/test/')))
+dataset = folder_data.register(workspace=ws, name=dataset_name)
+```
+
+### From outputs using `OutputFileDatasetConfig`
+```python
+from azureml.core import ScriptRunConfig
+from azureml.data import OutputFileDatasetConfig
+
+output_data = OutputFileDatasetConfig(
+    destination=(datastore, "test_path/{run-id}"),
```

Comment:
Same again, lets use a generic path. Note: the { } assume you are using an f string f"", which this example doesn't. I'd suggest

destination=(datastore, "path/to/output")